modules/ducktests/src/main/java/org/apache/ignite/internal/ducktest/tests/dns_failure_test/BlockingNameService.java

```diff
@@ -71,27 +69,11 @@ private static void installJdk11() throws Exception {
         System.out.println("Installed DnsBlocker as main NameService to JVM");
     }
 
-    /** Installs {@code BlockingNameService} as main {@code NameService} to JVM8. */
```

Comment:
I think jdk8 support should be preserved in test as such, since 'master' ducktests are supposed to work with any old ignite version.  Not only with the most recent one.

It's ok to make use jdk11 as default for testing via docker but if needed tests should work with jdk8 as well.